src/ashell/shell-state.c

```diff
@@ -578,17 +582,42 @@ int32_t ashell_check_control(const char *buf, uint32_t len)
     return RET_OK;
 }
 
+int32_t ashell_set_bootcfg(char *buf)
+{
+    if (!fs_exist(buf)) {
+        comms_println("File passed to cfg doesn't exist\n");
+        return RET_ERROR;
+    }
+
+    fs_file_t *file = fs_open_alloc("boot.cfg", "w+");
+    if (!file) {
+        comms_println("Failed to create boot.cfg file");
+        return RET_ERROR;
+    }
+
+    ssize_t written = fs_write(file, buf, strlen(buf));
+    if (written == 0) {
+        comms_println("Failed to write boot.cfg file");
+        fs_close_alloc(file);
+        return RET_OK;
+    }
+
+    fs_close_alloc(file);
+    return RET_OK;
+}
+
 #define ASHELL_COMMAND(name,syntax,cmd) {name, syntax, cmd}
 
 static const struct ashell_cmd commands[] =
 {
     ASHELL_COMMAND("help",  "This help", ashell_help),
-    ASHELL_COMMAND("eval",  "Evaluate JavaScript in real time"                ,ashell_js_immediate_mode),
+    ASHELL_COMMAND("eval",  "Evaluate JavaScript in real time"               ,ashell_js_immediate_mode),
     ASHELL_COMMAND("clear", "Clear the terminal screen"                      ,ashell_clear),
     ASHELL_COMMAND("load",  "[FILE] Saves the input text into a file"        ,ashell_read_data),
     ASHELL_COMMAND("run",   "[FILE] Runs the JavaScript program in the file" ,ashell_run_javascript),
     ASHELL_COMMAND("parse", "[FILE] Check if the JS syntax is correct"       ,ashell_parse_javascript),
     ASHELL_COMMAND("stop",  "Stops current JavaScript execution"             ,ashell_stop_javascript),
+    ASHELL_COMMAND("bootcfg", "[FILE] Set the file that should run at boot"  ,ashell_set_bootcfg),
```

Comment:
Not a required change but actually 'boot' would be fine and would fit here better, not requiring a bunch of indent adjustment.